### ISSUE TITLE
smaller EMP

### ISSUE THREAD

**Issue by bruhmoment39** [studio-minus/ppg-bugs-and-requests#1171]
The EMP right now has a giant range and it would be nice to have a feature to make it smaller or just have an EMP the size of a grenade blast.
**Feature purpose**
Describe the purpose here. What made you request this feature? What is its purpose? Are you sure it doesn't already exist in some form?
This would help if you were trying to make an EMP missile or such that needs to take out a vehicle but doesn't want to blast the whole mf room.

**Comment by DindinYT37** [comment]
Please use the template.

**Comment by bruhmoment39** [comment]
> Please use the template.

literally did

**Comment by DindinYT37** [comment]
> > Please use the template.
> 
> literally did

See no headers, which renders the template basically useless.

**Comment by bruhmoment39** [comment]
oh no i cant take two seconds to find out what the category is!!!!!!!!

**Comment by DindinYT37** [comment]
> oh no i cant take two seconds to find out what the category is!!!!!!!!

Oh no, I can't take two seconds to use the template/headers. What's your point.

**Comment by ppgplayer** [comment]
> > oh no i cant take two seconds to find out what the category is!!!!!!!!
> 
> Oh no, I can't take two seconds to use the template/headers. What's your point.

The point is nobody gives a shit Dindin, he used the damn template now leave the guy alone.

**Comment by DindinYT37** [comment]
> > > oh no i cant take two seconds to find out what the category is!!!!!!!!
> > 
> > Oh no, I can't take two seconds to use the template/headers. What's your point.
> 
> The point is nobody gives a shit Dindin, he used the damn template now leave the guy alone.

I don’t care about your opinion, and you know that if I wouldn't tell them, zooi would.

**Comment by ppgplayer** [comment]
> > > > oh no i cant take two seconds to find out what the category is!!!!!!!!
> > > 
> > > 
> > > Oh no, I can't take two seconds to use the template/headers. What's your point.
> > 
> > 
> > The point is nobody gives a shit Dindin, he used the damn template now leave the guy alone.
> 
> I don’t care about your opinion, and you know that if I wouldn't tell them, zooi would.

I don't really care for yours much either. Cut it out.

**Comment by DindinYT37** [comment]
> > > > > oh no i cant take two seconds to find out what the category is!!!!!!!!
> > > > 
> > > > 
> > > > Oh no, I can't take two seconds to use the template/headers. What's your point.
> > > 
> > > 
> > > The point is nobody gives a shit Dindin, he used the damn template now leave the guy alone.
> > 
> > 
> > I don’t care about your opinion, and you know that if I wouldn't tell them, zooi would.
> 
> I don't really care for yours much either. Cut it out.

No. You're not someone to give me orders, it's not my fault you're getting annoyed because of someone else's business, and you know very well that if I wouldn't tell them, zooi would anyway. Also, you could always just block me and not see my replies instead of just getting annoyed for no reason.
